<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>952</width>
        <height>705</height>
      </rect>
    </property>
    <property name="font">
      <font>
        <family>SF Pro</family>
        <pointsize>12</pointsize>
      </font>
    </property>
    <property name="windowTitle">
      <string>LocationsFITTool</string>
    </property>
    <property name="styleSheet">
      <string notr="true">QWidget {
        font-family: &quot;SF Pro&quot;;
        font-size: 12pt;
        }

        QTextEdit {
        font-family: &quot;SF Mono&quot;;
        font-size: 11pt;
        }
</string>
    </property>
    <property name="dockOptions">
      <set>QMainWindow::DockOption::AllowTabbedDocks|QMainWindow::DockOption::AnimatedDocks</set>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
          <layout class="QHBoxLayout" name="file_btns_layout">
            <item>
              <widget class="QPushButton" name="import_locations_fit_btn">
                <property name="text">
                  <string>Import Locations.fit</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="import_gpx_btn">
                <property name="text">
                  <string>Import GPX</string>
                </property>
              </widget>
            </item>
            <item>
              <spacer name="horizontalSpacer">
                <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>40</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
            <item>
              <widget class="QPushButton" name="save_locations_fit_btn">
                <property name="text">
                  <string>Save Locations.fit</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
              <widget class="QGroupBox" name="waypoint_group_box">
                <property name="title">
                  <string>Waypoints</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                    <layout class="QVBoxLayout" name="waypoint_area">
                      <property name="rightMargin">
                        <number>0</number>
                      </property>
                      <item>
                        <widget class="QTableWidget" name="waypoint_table" />
                      </item>
                      <item>
                        <layout class="QHBoxLayout" name="wpt_btns_layout">
                          <item>
                            <spacer name="horizontalSpacer_2">
                              <property name="orientation">
                                <enum>Qt::Orientation::Horizontal</enum>
                              </property>
                              <property name="sizeHint" stdset="0">
                                <size>
                                  <width>40</width>
                                  <height>20</height>
                                </size>
                              </property>
                            </spacer>
                          </item>
                          <item>
                            <widget class="QPushButton" name="add_wpt_btn">
                              <property name="text">
                                <string>Add</string>
                              </property>
                            </widget>
                          </item>
                          <item>
                            <widget class="QPushButton" name="delete_wpt_btn">
                              <property name="text">
                                <string>Delete</string>
                              </property>
                            </widget>
                          </item>
                          <item>
                            <widget class="QPushButton" name="delete_all_wpts_btn">
                              <property name="text">
                                <string>Delete All</string>
                              </property>
                            </widget>
                          </item>
                        </layout>
                      </item>
                    </layout>
                  </item>
                </layout>
              </widget>
            </item>
            <item>
              <widget class="QGroupBox" name="device_group_box">
                <property name="title">
                  <string>Device</string>
                </property>
                <property name="flat">
                  <bool>false</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                    <layout class="QVBoxLayout" name="device_area">
                      <item>
                        <widget class="QLabel" name="device_info_lbl">
                          <property name="text">
                            <string>-</string>
                          </property>
                          <property name="alignment">
                            <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignTop</set>
                          </property>
                          <property name="wordWrap">
                            <bool>true</bool>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </item>
                </layout>
              </widget>
            </item>
          </layout>
        </item>
      </layout>
    </widget>
    <widget class="QDockWidget" name="log_dock">
      <property name="enabled">
        <bool>true</bool>
      </property>
      <attribute name="dockWidgetArea">
        <number>8</number>
      </attribute>
      <widget class="QWidget" name="log_dock_contents">
        <property name="minimumSize">
          <size>
            <width>0</width>
            <height>100</height>
          </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
          <item>
            <widget class="QTextEdit" name="log_textedit" />
          </item>
        </layout>
      </widget>
    </widget>
    <widget class="QMenuBar" name="menuBar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>578</width>
          <height>24</height>
        </rect>
      </property>
      <widget class="QMenu" name="menuFile">
        <property name="title">
          <string>File</string>
        </property>
        <addaction name="import_locations_fit_action" />
        <addaction name="import_gpx_action" />
        <addaction name="separator" />
        <addaction name="save_locations_fit_action" />
        <addaction name="separator" />
      </widget>
      <widget class="QMenu" name="menuWaypoint">
        <property name="title">
          <string>Waypoint</string>
        </property>
        <addaction name="add_wpt_action" />
        <addaction name="delete_wpt_action" />
        <addaction name="delete_all_wpts_action" />
      </widget>
      <widget class="QMenu" name="menuView">
        <property name="title">
          <string>View</string>
        </property>
        <addaction name="toggle_debug_log_action" />
      </widget>
      <widget class="QMenu" name="menuDevice">
        <property name="title">
          <string>Device</string>
        </property>
        <addaction name="scan_for_devices_action" />
      </widget>
      <addaction name="menuFile" />
      <addaction name="menuWaypoint" />
      <addaction name="menuView" />
      <addaction name="menuDevice" />
    </widget>
    <widget class="QStatusBar" name="status_bar" />
    <action name="import_locations_fit_action">
      <property name="text">
        <string>Import Locations.fit</string>
      </property>
      <property name="toolTip">
        <string>Import Waypoints from Locations.fit file</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+O</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="import_gpx_action">
      <property name="text">
        <string>Import GPX</string>
      </property>
      <property name="toolTip">
        <string>Import Waypoints from GPX file</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+L</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="save_locations_fit_action">
      <property name="text">
        <string>Save Locations.fit</string>
      </property>
      <property name="toolTip">
        <string>Save Waypoints to new Locations.fit file</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+S</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="add_wpt_action">
      <property name="text">
        <string>Add Waypoint</string>
      </property>
      <property name="shortcut">
        <string>Ctrl++</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="delete_wpt_action">
      <property name="text">
        <string>Delete Waypoint</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+Backspace</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="delete_all_wpts_action">
      <property name="text">
        <string>Delete All Waypoints</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+Shift+Backspace</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="toggle_debug_log_action">
      <property name="checkable">
        <bool>true</bool>
      </property>
      <property name="text">
        <string>Toggle Debug Log</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+D</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="scan_for_devices_action">
      <property name="checkable">
        <bool>true</bool>
      </property>
      <property name="text">
        <string>Scan For Devices</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+C</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
  </widget>
  <resources />
  <connections>
    <connection>
      <sender>toggle_debug_log_action</sender>
      <signal>toggled(bool)</signal>
      <receiver>log_dock</receiver>
      <slot>setVisible(bool)</slot>
      <hints>
        <hint type="sourcelabel">
          <x>-1</x>
          <y>-1</y>
        </hint>
        <hint type="destinationlabel">
          <x>399</x>
          <y>480</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>save_locations_fit_btn</sender>
      <signal>clicked()</signal>
      <receiver>save_locations_fit_action</receiver>
      <slot>trigger()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>717</x>
          <y>27</y>
        </hint>
        <hint type="destinationlabel">
          <x>-1</x>
          <y>-1</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>import_gpx_btn</sender>
      <signal>clicked()</signal>
      <receiver>import_gpx_action</receiver>
      <slot>trigger()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>225</x>
          <y>27</y>
        </hint>
        <hint type="destinationlabel">
          <x>-1</x>
          <y>-1</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>import_locations_fit_btn</sender>
      <signal>clicked()</signal>
      <receiver>import_locations_fit_action</receiver>
      <slot>trigger()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>87</x>
          <y>27</y>
        </hint>
        <hint type="destinationlabel">
          <x>-1</x>
          <y>-1</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>add_wpt_btn</sender>
      <signal>clicked()</signal>
      <receiver>add_wpt_action</receiver>
      <slot>trigger()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>137</x>
          <y>544</y>
        </hint>
        <hint type="destinationlabel">
          <x>-1</x>
          <y>-1</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>delete_all_wpts_btn</sender>
      <signal>clicked()</signal>
      <receiver>delete_all_wpts_action</receiver>
      <slot>trigger()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>662</x>
          <y>544</y>
        </hint>
        <hint type="destinationlabel">
          <x>-1</x>
          <y>-1</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>delete_wpt_btn</sender>
      <signal>clicked()</signal>
      <receiver>delete_wpt_action</receiver>
      <slot>trigger()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>399</x>
          <y>544</y>
        </hint>
        <hint type="destinationlabel">
          <x>-1</x>
          <y>-1</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>